client/account-recovery/index.js

```diff
@@ -13,15 +13,17 @@ import {
 } from './controller';
 import config from 'config';
 
+import { ACCOUNT_RECOVERY_ROUTES } from './constants';
```

Comment:
The `ACCOUNT_RECOVERY_` noise could be cut down by changing this to something like:

```js
import { ACCOUNT_RECOVERY_ROUTES as ROUTES } from './constants';
```